Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -29,44 +30,38 @@ type IndicatorSize = number | 'small' | 'large';
 
 type IOSProps = $ReadOnly<{|
   /**
-   * Whether the indicator should hide when not animating (true by default).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
-   */
+    Whether the indicator should hide when not animating.
+
+    @platform ios
+  */
   hidesWhenStopped?: ?boolean,
 |}>;
 type Props = $ReadOnly<{|
   ...ViewProps,
   ...IOSProps,
 
   /**
-   * Whether to show the indicator (true, the default) or hide it (false).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#animating
+   	Whether to show the indicator (`true`) or hide it (`false`).
    */
   animating?: ?boolean,
 
   /**
-   * The foreground color of the spinner (default is gray).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#color
-   */
+    The foreground color of the spinner.
+
+    @default {@platform android} `null` (system accent default color)
+    @default {@platform ios} '#999999'
+  */
   color?: ?ColorValue,
 
   /**
-   * Size of the indicator (default is 'small').
-   * Passing a number to the size prop is only supported on Android.
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#size
-   */
+    Size of the indicator.
+
+    @type enum(`'small'`, `'large'`)
+    @type {@platform android} number
```

Comment:
**Long term comment - no need to fix before merging**

This use of multiple `@type` annotations is pretty clever!

I think in the long run we should look into annotating the Flow types themselves, e.g. maybe something like:

```js
type IndicatorSize = 'small' | 'large' | /** @platform android **/ number;
```

Or using Flow types directly:
```js
// A utility type which the docs generator will treat specially
type RN$PlatformSpecific<PlatformT, T> = T;

type IndicatorSize = 'small' | 'large' | RN$PlatformSpecific<'android', number>;
```

The latter has the added benefit that, in the future, we can reimplement `RN$PlatformSpecific` to actually enforce the correct types per platform.

Reply:
Yes. Thanks 😃